src/overlays/actors/ovl_En_Wiz/z_en_wiz.c

```diff
@@ -30,199 +78,1465 @@ const ActorInit En_Wiz_InitVars = {
+        this->actor.colChkInfo.damageTable = &sFireWizrobeDamageTable;
+        this->actor.colChkInfo.health = 8;
+        this->actor.flags &= ~ACTOR_FLAG_100000;
+    } else {
+        this->actor.colChkInfo.damageTable = &sIceWizrobeDamageTable;
+        this->actor.colChkInfo.health = 6;
+    }
+
+    if ((this->switchFlag >= 0) && (Flags_GetSwitch(play, this->switchFlag))) {
+        Actor_MarkForDeath(&this->actor);
+    } else {
+        this->actor.hintId = 0x4B;
+        this->currentPlatformIndex = 777;
+
+        // Setting the radius and scale to zero here effectively disables all of the ghost colliders.
+        this->ghostColliders.elements[0].dim.modelSphere.radius = 0;
+        this->ghostColliders.elements[0].dim.scale = 0.0f;
+        this->ghostColliders.elements[0].dim.modelSphere.center.x = 0;
+        this->ghostColliders.elements[0].dim.modelSphere.center.y = 0;
+        this->ghostColliders.elements[0].dim.modelSphere.center.z = 0;
+
+        this->actionFunc = EnWiz_StartIntroCutscene;
+    }
+}
+
+void EnWiz_Destroy(Actor* thisx, PlayState* play) {
+    s32 pad;
+    EnWiz* this = THIS;
+
+    Collider_DestroyCylinder(play, &this->collider);
+    Collider_DestroyJntSph(play, &this->ghostColliders);
+    if (this->type != EN_WIZ_TYPE_FIRE_NO_MINI_BOSS_BGM) {
+        func_801A2ED8();
+    }
+}
+
+void EnWiz_ChangeAnim(EnWiz* this, s32 animIndex, s32 updateGhostAnim) {
+    this->endFrame = Animation_GetLastFrame(sAnimations[animIndex]);
+    Animation_Change(&this->skelAnime, sAnimations[animIndex], 1.0f, 0.0f, this->endFrame, sAnimationModes[animIndex],
+                     -2.0f);
+    if (updateGhostAnim) {
+        Animation_Change(&this->ghostSkelAnime, sAnimations[animIndex], 1.0f, 0.0f, this->endFrame,
+                         sAnimationModes[animIndex], -2.0f);
+    }
+}
+
+/**
+ * Responsible for moving the camera around and making the Wizrobe run in circles during the intro cutscene.
+ */
+void EnWiz_HandleIntroCutscene(EnWiz* this, PlayState* play) {
+    Camera* camera;
+    Vec3f targetEye;
+    Vec3f targetAt;
+
+    if (this->introCutsceneState < EN_WIZ_INTRO_CS_DISAPPEAR) {
+        camera = Play_GetCamera(play, this->subCamId);
+        switch (this->introCutsceneState) {
+            case EN_WIZ_INTRO_CS_NOT_STARTED:
+                this->introCutsceneTimer = 100;
+                this->introCutsceneCameraAngle = this->actor.world.rot.y;
+                this->introCutsceneState++;
+                break;
+
+            case EN_WIZ_INTRO_CS_CAMERA_MOVE_TO_PLATFORM:
+                Math_Vec3f_Copy(&targetEye, &this->actor.world.pos);
+                Math_Vec3f_Copy(&targetAt, &this->actor.world.pos);
+                targetEye.x += Math_SinS(this->introCutsceneCameraAngle) * 200.0f;
+                targetEye.y += 100.0f;
+                targetEye.z += Math_CosS(this->introCutsceneCameraAngle) * 200.0f;
+                targetAt.y += 80.0f;
```

Comment:
```suggestion
                eyeNext.z += Math_CosS(this->introCutsceneCameraAngle) * 200.0f;
                atNext.y += 80.0f;
```
One of the reasons we avoid "target" with camera is that "target" means something specific i.e. the z-targeted actor, which would be the wizrobe. I'm not sure we'll always stick with that notation, I'm not the biggest fan tbh. But it's consistent for now